input/fsh/models.fsh

```diff
@@ -30,14 +30,13 @@ criteria are defined by FHIRPath expressions.
   Name of the view definition, must be in a database-friendly format.
 """
 * name obeys sql-name
-* version 0..1 string "Business version of the view definition"
```

Comment:
Removed a couple things since they are redundant with the Meta structure content